### PR TITLE
ignition-rendering5: update to latest commit

### PR DESCRIPTION
**Pull request by adlarkin**
Need latest `rendering5` features for https://github.com/ignitionrobotics/ign-gazebo/pull/460

Signed-off-by: Ashton Larkin <ashton@openrobotics.org>

### CODE REVIEW

**Comment by scpeters** [comment]
build bottle

File: Formula/ignition-rendering5.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
   desc "Rendering library for robotics applications"
   homepage "https://github.com/ignitionrobotics/ign-rendering"
-  url "https://github.com/ignitionrobotics/ign-rendering/archive/d18dd1c6540b2aaa581a3f48ce96022d7ba19bda.tar.gz"
-  version "4.999.999~0~20201028~d18dd1"
-  sha256 "4c97c5eb65b3c939db06176b5d1b3728af69ce17a0bb7ba522fcadcbac6b5c2b"
+  url "https://github.com/ignitionrobotics/ign-rendering/archive/d4406e1d98fba7b1362ce8ee52056cfd4ecb0bdb.tar.gz"
+  version "4.999.999~0~20201130~d4406e"
+  sha256 "781a633371a2525ff6f622a275456b72ec40467726dd452deba82aa28422be94"
   license "Apache-2.0"
   revision 1
```
Comment (chapulina) [inline Formula/ignition-rendering5.rb:8]: The bottle build [failed](https://build.osrfoundation.org/job/generic-release-homebrew_triggered_bottle_builder/label=osx_mojave/108/console) with:

```
==> brew audit osrf/simulation/ignition-rendering5 --online --git --skip-style
==> FAILED
Error: 1 problem in 1 formula detected
osrf/simulation/ignition-rendering5:
  * 'revision 1' should be removed
```

I think this line needs to be removed? I'm not sure why it was ok before though...

Comment (scpeters) [reply Formula/ignition-rendering5.rb:8]: our `brew test-bot` CI doesn't do very much; it mainly checks for syntax errors, but it doesn't call `brew audit` on changed formulae (see https://github.com/osrf/homebrew-simulation/issues/1139)

so yes, this line should be removed

Comment (adlarkin) [reply Formula/ignition-rendering5.rb]: Removed in https://github.com/osrf/homebrew-simulation/pull/1214/commits/85696895b823d5c970b0a191af665ace6aea7a05

**Comment by chapulina** [comment]
build bottle

File: Formula/ignition-rendering5.rb (hunk 1 above)
Comment (scpeters) [reply Formula/ignition-rendering5.rb]: > ==> brew audit osrf/simulation/ignition-rendering5 --online --git --skip-style
> ==> FAILED
> Error: 1 problem in 1 formula detected
> osrf/simulation/ignition-rendering5:
>   * 'revision 1' should be removed

I think our current CI calls `brew audit` via test-bot but not `brew audit --online --git`

**Pull request merged**